app/build.gradle.kts

```
@@ -60,6 +60,9 @@ dependencies {
     implementation ("androidx.navigation:navigation-ui-ktx:2.5.3")
     implementation ("androidx.fragment:fragment-ktx:1.5.6")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4")
+    implementation("androidx.room:room-runtime:2.6.1")
```

минор: можно сделать аналогично moxy: объявить переменную с версией, и в трёх строчках использовать только её
```suggestion
    implementation("androidx.room:room-runtime:$roomVersion")
```